yjit/src/codegen.rs

```diff
@@ -147,14 +148,12 @@ jit_obj_info_dump(codeblock_t *cb, x86opnd_t opnd) {
     call_ptr(cb, REG0, (void *)rb_obj_info_dump);
     pop_regs(cb);
 }
-
+*/
 // Get the index of the next instruction
-static uint32_t
-jit_next_insn_idx(jitstate_t *jit)
+fn jit_next_insn_idx(jit: &JITState) -> u32
 {
-    return jit->insn_idx + insn_len(jit_get_opcode(jit));
+    (jit.insn_idx + insn_len(jit.get_opcode())).try_into().unwrap()
```

Comment:
Hopefully we don't need `try_into().unwrap()` now that everything is u32? 🤞

Reply:
I've remove a bunch of that. Let me know if you see anything else that should go.